packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -889,6 +891,203 @@ describe("handleNewBooking", () => {
+              },
+              update: {
+                uid: "UPDATED_MOCK_ID",
+                iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
+              },
+            });
+
+            const mockBookingData = getMockRequestDataForBooking({
+              data: {
+                eventTypeId: 1,
+                rescheduleUid: uidOfBookingToBeRescheduled,
+                start: `${plus1DateString}T04:00:00.000Z`,
+                end: `${plus1DateString}T04:15:00.000Z`,
+                responses: {
+                  email: booker.email,
+                  name: booker.name,
+                  location: { optionValue: "", value: OrganizerDefaultConferencingAppType },
+                },
+              },
+            });
+
+            const { req } = createMockNextJsRequest({
+              method: "POST",
+              body: mockBookingData,
+            });
+
+            const createdBooking = await handleNewBooking(req);
+
+            expect(createdBooking.responses).toContain({
+              email: booker.email,
+              name: booker.name,
+            });
+
+            await expectBookingInDBToBeRescheduledFromTo({
+              from: {
+                uid: uidOfBookingToBeRescheduled,
+              },
+              to: {
+                description: "",
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                uid: createdBooking.uid!,
+                eventTypeId: mockBookingData.eventTypeId,
+                // Rescheduled booking sill stays in pending state
+                status: BookingStatus.PENDING,
+                location: BookingLocations.GoogleMeet,
+                responses: expect.objectContaining({
+                  email: booker.email,
+                  name: booker.name,
+                }),
+                references: [
+                  {
+                    type: appStoreMetadata.googlecalendar.type,
+                    uid: "MOCK_ID",
+                    meetingId: "MOCK_ID",
+                    meetingPassword: "MOCK_PASSWORD",
+                    meetingUrl: "https://UNUSED_URL",
+                    externalCalendarId: "MOCK_EXTERNAL_CALENDAR_ID",
+                  },
+                ],
+              },
+            });
+
+            expectWorkflowToBeTriggered({ emails, organizer });
+
+            expectBookingRequestedEmails({
+              booker,
+              organizer,
+              emails,
+            });
+
+            expectBookingRequestedWebhookToHaveBeenFired({
+              booker,
+              organizer,
+              location: OrganizerDefaultConferencingAppType,
+              subscriberUrl,
+              eventType: scenarioData.eventTypes[0],
+            });
+
+            expectSuccessfulCalendarEventDeletionInCalendar(calendarMock, {
+              externalCalendarId: "MOCK_EXTERNAL_CALENDAR_ID",
+              calEvent: {},
+              uid: "MOCK_ID",
+            });
```

Comment:
These assertions seem to be passing in main as well for me. Could you recheck that if we ever have this issue again this test would catch it.